### PR TITLE
[Docs | Rule Tuning] Add blog references to rules

### PR DESCRIPTION
**Pull request by Mikaayenson**
# Pull Request

*Issue link(s)*: N/A

## Summary - What I changed

- This PR adds our blog content as references to our prebuilt rules. 
- Noticed some auto formatting was added on toml_save.

## How To Test

- Check out any of the blogs added as a reference and double check that the rule is mentioned in the blog.

## Checklist

- [x] Added a label for the type of pr: `bug`, `enhancement`, `schema`, `Rule: New`, `Rule: Deprecation`, `Rule: Tuning`, `Hunt: New`, or `Hunt: Tuning` so guidelines can be generated
- [ ] Added the `meta:rapid-merge` label if planning to merge within 24 hours
- [ ] Secret and sensitive material has been managed correctly
- [ ] Automated testing was updated or added to match the most common scenarios
- [ ] Documentation and comments were added for features that require explanation

### CODE REVIEW

**Comment by protectionsmachine** [comment]
## Rule: Tuning - Guidelines

These guidelines serve as a reminder set of considerations when tuning an existing rule.

### Documentation and Context

- [ ] Detailed description of the suggested changes.
- [ ] Provide example JSON data or screenshots.
- [ ] Provide evidence of reducing benign events mistakenly identified as threats (False Positives).
- [ ] Provide evidence of enhancing detection of true threats that were previously missed (False Negatives).
- [ ] Provide evidence of optimizing resource consumption and execution time of detection rules (Performance).
- [ ] Provide evidence of specific environment factors influencing customized rule tuning (Contextual Tuning).
- [ ] Provide evidence of improvements made by modifying sensitivity by changing alert triggering thresholds (Threshold Adjustments).
- [ ] Provide evidence of refining rules to better detect deviations from typical behavior (Behavioral Tuning).
- [ ] Provide evidence of improvements of adjusting rules based on time-based patterns (Temporal Tuning).
- [ ] Provide reasoning of adjusting priority or severity levels of alerts (Severity Tuning).
- [ ] Provide evidence of improving quality integrity of our data used by detection rules (Data Quality).
- [ ] Ensure the tuning includes necessary updates to the release documentation and versioning.

### Rule Metadata Checks

- [ ] `updated_date` matches the date of tuning PR merged.
- [ ] `min_stack_version` should support the widest stack versions.
- [ ] `name` and `description` should be descriptive and not include typos.
- [ ] `query` should be inclusive, not overly exclusive. Review to ensure the original intent of the rule is maintained.

### Testing and Validation

- [ ] Validate that the tuned rule's performance is satisfactory and does not negatively impact the stack.
- [ ] Ensure that the tuned rule has a low false positive rate.

File: rules/integrations/google_workspace/collection_google_drive_ownership_transferred_via_google_workspace.toml [hunk 1]
```diff
@@ -76,7 +76,10 @@ The Google Workspace Fleet integration, Filebeat module, or similarly structured
 - See the following references for further information:
   - https://support.google.com/a/answer/7061566
   - https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-google_workspace.html"""
-references = ["https://support.google.com/a/answer/1247799?hl=en"]
+references = [
+    "https://support.google.com/a/answer/1247799?hl=en",
+    "https://www.elastic.co/security-labs/google-workspace-attack-surface-part-two",
```
Comment (terrancedejesus) [inline rules/integrations/google_workspace/collection_google_drive_ownership_transferred_via_google_workspace.toml:81]: https://www.elastic.co/security-labs/google-workspace-attack-surface-part-one
https://www.elastic.co/security-labs/google-workspace-attack-surface-part-two

I'd just add these to each rule for Google Workspace then. They are referenced in docs.

File: rules/integrations/okta/credential_access_attempted_bypass_of_okta_mfa.toml [hunk 2]
```diff
@@ -58,6 +58,8 @@ references = [
     "https://developer.okta.com/docs/reference/api/system-log/",
     "https://developer.okta.com/docs/reference/api/event-types/",
     "https://www.elastic.co/security-labs/testing-okta-visibility-and-detection-dorothy",
+    "https://www.elastic.co/security-labs/monitoring-okta-threats-with-elastic-security",
```
Comment (terrancedejesus) [inline rules/integrations/okta/credential_access_attempted_bypass_of_okta_mfa.toml:61]: I'd add these to all Okta, along with https://www.elastic.co/security-labs/starter-guide-to-understanding-okta

**Review by terrancedejesus — APPROVED** [review]
Added a note for Okta and GWS.

File: rules/windows/command_and_control_common_webservices.toml [hunk 3]
```diff
@@ -128,10 +127,18 @@ This rule looks for processes outside known legitimate program locations communi
 - Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
 - Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the mean time to respond (MTTR).
 """
+references = ["https://www.elastic.co/security-labs/operation-bleeding-bear"]
```
Comment (Samirbous) [inline rules/windows/command_and_control_common_webservices.toml:130]: siestagraph uses a websvc domain for C2 `graph.microsoft.com` (covered in this rule)

```suggestion
references = [
"https://www.elastic.co/security-labs/operation-bleeding-bear", 
"https://www.elastic.co/security-labs/siestagraph-new-implant-uncovered-in-asean-member-foreign-ministry"
]
```

File: rules/windows/defense_evasion_defender_exclusion_via_powershell.toml [hunk 4]
```diff
@@ -70,6 +70,8 @@ Microsoft Windows Defender is an antivirus product built into Microsoft Windows.
 """
 references = [
     "https://www.bitdefender.com/files/News/CaseStudies/study/400/Bitdefender-PR-Whitepaper-MosaicLoader-creat5540-en-EN.pdf",
+    "https://www.elastic.co/security-labs/elastic-security-uncovers-blister-malware-campaign",
+    "https://www.elastic.co/security-labs/operation-bleeding-bear",
 ]
```
Comment (Samirbous) [inline rules/windows/defense_evasion_defender_exclusion_via_powershell.toml:75]: ghostengine also disables Windows defender: 

```suggestion
references = [
    "https://www.bitdefender.com/files/News/CaseStudies/study/400/Bitdefender-PR-Whitepaper-MosaicLoader-creat5540-en-EN.pdf",
    "https://www.elastic.co/security-labs/elastic-security-uncovers-blister-malware-campaign",
    "https://www.elastic.co/security-labs/operation-bleeding-bear",
    "https://www.elastic.co/security-labs/invisible-miners-unveiling-ghostengine"
]
```

File: rules/windows/defense_evasion_disabling_windows_defender_powershell.toml [hunk 5]
```diff
@@ -58,6 +58,7 @@ This rule monitors the execution of commands that can tamper the Windows Defende
 """
 references = [
     "https://docs.microsoft.com/en-us/powershell/module/defender/set-mppreference?view=windowsserver2019-ps",
+    "https://www.elastic.co/security-labs/operation-bleeding-bear",
```
Comment (Samirbous) [inline rules/windows/defense_evasion_disabling_windows_defender_powershell.toml:61]:
```suggestion
    "https://www.elastic.co/security-labs/operation-bleeding-bear", 
    "https://www.elastic.co/security-labs/invisible-miners-unveiling-ghostengine"
```

File: rules/windows/defense_evasion_from_unusual_directory.toml [hunk 6]
```diff
@@ -102,6 +102,7 @@ This rule identifies processes that are executed from suspicious default Windows
 - Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
 - Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the mean time to respond (MTTR).
 """
+references = ["https://www.elastic.co/security-labs/invisible-miners-unveiling-ghostengine"]
```
Comment (Samirbous) [inline rules/windows/defense_evasion_from_unusual_directory.toml:105]: Siestagraph executes from `Windows\\Tasks` covered in this rule : 

```suggestion
references = [
"https://www.elastic.co/security-labs/invisible-miners-unveiling-ghostengine", 
"https://www.elastic.co/security-labs/siestagraph-new-implant-uncovered-in-asean-member-foreign-ministry"
]
```

File: rules/windows/lateral_movement_direct_outbound_smb_connection.toml [hunk 7]
```diff
@@ -95,6 +98,9 @@ This rule looks for unexpected processes or LOLBins making network connections o
 - Determine the initial vector abused by the attacker and take action to prevent reinfection through the same vector.
 - Using the incident response data, update logging and audit policies to improve the mean time to detect (MTTD) and the mean time to respond (MTTR).
 """
+references = [
+    "https://www.elastic.co/security-labs/elastic-protects-against-data-wiper-malware-targeting-ukraine-hermeticwiper",
+]
```
Comment (Samirbous) [inline rules/windows/lateral_movement_direct_outbound_smb_connection.toml:103]:
```suggestion
references = [
    "https://www.elastic.co/security-labs/elastic-protects-against-data-wiper-malware-targeting-ukraine-hermeticwiper",
    "https://www.elastic.co/security-labs/hunting-for-lateral-movement-using-event-query-language"
]
```

**Review by Samirbous — APPROVED** [review]

**Review by Aegrah — APPROVED** [review]

**Review by eric-forte-elastic — APPROVED** [review]

File: rules/integrations/google_workspace/collection_google_drive_ownership_transferred_via_google_workspace.toml [hunk 8]
```diff
@@ -76,7 +76,10 @@ The Google Workspace Fleet integration, Filebeat module, or similarly structured
...
+]
```
Comment (Mikaayenson) [inline rules/integrations/google_workspace/collection_google_drive_ownership_transferred_via_google_workspace.toml:82]:
```suggestion
references = [
    "https://support.google.com/a/answer/1247799?hl=en",
    "https://www.elastic.co/security-labs/google-workspace-attack-surface-part-two",
    "https://www.elastic.co/security-labs/google-workspace-attack-surface-part-one",
    "https://www.elastic.co/security-labs/google-workspace-attack-surface-part-two"
]
```

File: rules/integrations/google_workspace/persistence_google_workspace_custom_admin_role_created.toml [hunk 9]
```diff
@@ -79,7 +79,10 @@ The Google Workspace Fleet integration, Filebeat module, or similarly structured
 - See the following references for further information:
   - https://support.google.com/a/answer/7061566
   - https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-google_workspace.html"""
-references = ["https://support.google.com/a/answer/2406043?hl=en"]
+references = [
+    "https://support.google.com/a/answer/2406043?hl=en",
+    "https://www.elastic.co/security-labs/google-workspace-attack-surface-part-two",
+]
```
Comment (Mikaayenson) [inline rules/integrations/google_workspace/persistence_google_workspace_custom_admin_role_created.toml:85]:
```suggestion
references = [
    "https://support.google.com/a/answer/2406043?hl=en",
    "https://www.elastic.co/security-labs/google-workspace-attack-surface-part-one",
    "https://www.elastic.co/security-labs/google-workspace-attack-surface-part-two",
]
```

File: rules/integrations/okta/credential_access_attempted_bypass_of_okta_mfa.toml [hunk 10]
```diff
@@ -58,6 +58,8 @@ references = [
...
+    "https://www.elastic.co/security-labs/okta-and-lapsus-what-you-need-to-know",
```
Comment (Mikaayenson) [inline rules/integrations/okta/credential_access_attempted_bypass_of_okta_mfa.toml:62]:
```suggestion
    "https://www.elastic.co/security-labs/monitoring-okta-threats-with-elastic-security",
    "https://www.elastic.co/security-labs/okta-and-lapsus-what-you-need-to-know",
    "https://www.elastic.co/security-labs/starter-guide-to-understanding-okta",
```

File: rules/integrations/okta/persistence_attempt_to_reset_mfa_factors_for_okta_user_account.toml [hunk 11]
```diff
@@ -28,6 +28,7 @@ references = [
     "https://developer.okta.com/docs/reference/api/system-log/",
     "https://developer.okta.com/docs/reference/api/event-types/",
     "https://www.elastic.co/security-labs/testing-okta-visibility-and-detection-dorothy",
+    "https://www.elastic.co/security-labs/monitoring-okta-threats-with-elastic-security",
```
Comment (Mikaayenson) [inline rules/integrations/okta/persistence_attempt_to_reset_mfa_factors_for_okta_user_account.toml:31]:
```suggestion
    "https://www.elastic.co/security-labs/monitoring-okta-threats-with-elastic-security",
    "https://www.elastic.co/security-labs/starter-guide-to-understanding-okta",
```

**Review by DefSecSentinel — APPROVED** [review]
LGTM

**Pull request merged**